src/core/runtime/NOELLE_APIs.c

```diff
@@ -48,13 +51,40 @@ extern DispatcherInfo NOELLE_HELIX_dispatcher_sequentialSegments(
                              void *,
                              void *,
                              void *,
+                             void *,
                              int64_t,
                              int64_t,
                              uint64_t *),
     void *env,
     void *loopCarriedArray,
     int64_t numCores,
-    int64_t numOfsequentialSegments);
+    int64_t numOfsequentialSegments,
+    int8_t useScylax);
+
+extern int NOELLE_Scylax_printf(void *scylaxData, const char *format, ...);
```

Comment:
Can we find a more descriptive name?

Reply:
See above re: scylaxPerThreadDataArg. Otherwise it could be GINO_parallel_printf etc (and even if it remains Scylax I'll change NOELLE to GINO in these).